src/main.c

```diff
@@ -50,7 +50,7 @@ static int __init wg_mod_init(void)
 	if (ret < 0)
 		goto err_netlink;
 
-	pr_info("WireGuard " WIREGUARD_VERSION " loaded. See www.wireguard.com for information.\n");
+	pr_info("WireGuard " WIREGUARD_VERSION " (Amnezia VPN) loaded. See www.wireguard.com for information.\n");
```

Comment:
AmneziaWG" WIREGUARD_VERSION "loaded

Reply:
pr_info("AmneziaWG" WIREGUARD_VERSION " loaded. See amnezia.org for information.\n");